frontend/src/metabase/browse/components/BrowseModels.tsx

```diff
@@ -0,0 +1,192 @@
+        <Box maw="24rem">{t`Models help curate data to make it easier to find answers to questions all in one place.`}</Box>
+      }
+      illustrationElement={
+        <Box mb=".5rem">
+          <img src={NoResults} />
+        </Box>
+      }
+    />
+  );
+};
+
+const ModelGroup = ({
+  models,
+  index,
+}: {
+  models: SearchResult[];
+  index: number;
+}) => {
+  const sortedModels = models.sort(sortModels);
+  const collection = models[0].collection;
+
+  /** This id is used by aria-labelledby */
+  const collectionHtmlId = `collection-${collection?.id ?? `index-${index}`}`;
+
+  return (
+    <>
+      <CollectionHeader
+        collection={models[0].collection}
+        key={collectionHtmlId}
+        id={collectionHtmlId}
+      />
+      {sortedModels.map(model => (
+        <ModelCell
+          model={model}
+          collectionHtmlId={collectionHtmlId}
+          key={`model-${model.id}`}
+        />
+      ))}
+    </>
+  );
+};
+
+interface ModelCellProps {
+  model: SearchResult;
+  style?: React.CSSProperties;
+  collectionHtmlId: string;
+}
+
+const ModelCell = ({ model, style, collectionHtmlId }: ModelCellProps) => {
+  const headingId = `heading-for-model-${model.id}`;
+
+  const lastEditorFullName =
+    model.last_editor_common_name ?? model.creator_common_name;
+  const timestamp = model.last_edited_at ?? model.created_at ?? "";
+
+  return (
+    <Link
+      aria-labelledby={`${collectionHtmlId} ${headingId}`}
+      key={model.id}
+      style={style}
+      to={Urls.model(model as unknown as Partial<Card>)}
+    >
+      <ModelCard>
+        <Title order={4} className="text-wrap" lh="1rem" mb=".5rem">
+          <MultilineEllipsified id={headingId}>
+            {model.name}
+          </MultilineEllipsified>
+        </Title>
+        <Text h="2rem" size="xs" mb="auto">
+          <MultilineEllipsified
+            tooltipMaxWidth="100%"
+            className={cx({ "text-light": !model.description })}
+          >
+            {model.description || "No description."}{" "}
+          </MultilineEllipsified>
+        </Text>
+        <LastEdited
+          lastEditorFullName={lastEditorFullName}
+          timestamp={timestamp}
+        />
+      </ModelCard>
+    </Link>
+  );
+};
+
+const CollectionHeader = ({
+  collection,
+  style,
+  id,
+}: {
+  collection?: Pick<Collection, "id" | "name"> | null;
+  style?: React.CSSProperties;
+  id: string;
+}) => {
+  const MaybeLink = ({ children }: { children: React.ReactNode }) =>
+    collection ? (
+      <Group grow noWrap>
+        <CollectionHeaderLink to={Urls.collection(collection)}>
+          {children}
+        </CollectionHeaderLink>
+      </Group>
+    ) : (
+      <>{children}</>
+    );
+  return (
+    <CollectionHeaderContainer id={id} role="heading" style={style}>
+      <MaybeLink>
+        <Group spacing=".33rem">
+          <Icon name="folder" color={"text-dark"} size={16} />
+          <Text>{collection?.name || "Untitled collection"}</Text>
```

Comment:
Here, we need to handle "Our Analytics" with the `metabase/collections/utils` > `isRootCollection()`


![Screen Shot 2024-01-25 at 1 44 45 PM](https://github.com/metabase/metabase/assets/30528226/1cc02230-a754-43ae-8dc4-3e9ec95b348a)
![Screen Shot 2024-01-25 at 1 44 35 PM](https://github.com/metabase/metabase/assets/30528226/03512f3b-c1e8-4cfd-94ff-6882682cf712)

Reply:
Fixed

Btw, n.b.: https://github.com/metabase/metabase/blob/models-in-browse-data/frontend/src/metabase/query_builder/components/view/QuestionDataSource.jsx#L130